src/db/db_build.js

```diff
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const connect = require('./db_connections');
-// eslint-disable-next-line quotes
 const buildScript = fs.readFileSync(`${__dirname}/db_build.sql`, `utf8`);
```

Comment:
The backticks for `utf8` are unnecessary as that is just a string.

Reply:
Fixed